docker/test/stateless/Dockerfile

```diff
@@ -53,13 +52,15 @@ RUN ln -snf /usr/share/zoneinfo/$TZ /etc/localtime && echo $TZ > /etc/timezone
 ENV NUM_TRIES=1
 ENV MAX_RUN_TIME=0
 
+# See usage of the MINIO_VERSION in setup_minio.sh
+ENV MINIO_VERSION=2022-01-03T18-22-58Z
 ARG TARGETARCH
 
 # Download Minio-related binaries
 RUN arch=${TARGETARCH:-amd64} \
-    && if [ "$arch" = "amd64" ] ; then wget "https://dl.min.io/server/minio/release/linux-${arch}/archive/minio-20220103182258.0.0.x86_64.rpm"; else wget "https://dl.min.io/server/minio/release/linux-${arch}/archive/minio-20220103182258.0.0.aarch64.rpm" ; fi \
+    && wget "https://dl.min.io/server/minio/release/linux-${arch}/archive/minio.RELEASE.${MINIO_VERSION}" -O ./minio \
     && wget "https://dl.min.io/client/mc/release/linux-${arch}/mc" \
```

Comment:
Should we use specific version of `mc` as well? (whatever it is)

Reply:
it's unfortunate that it has different version `2022-01-05T23-52-51Z`, but it makes sense pinning it for sure